scripts/migrations/write_prefs_to_store.py

```diff
@@ -34,7 +34,7 @@ def copy_preferences_to_store(keys, verbose: bool = False) -> list[str]:
         try:
             if verbose:
                 print(f"Writing {key} to store...")
-            username = key.split('/')[-1]
+            username = key.split('/')[-2]
```

Comment:
The `key` here is a preference key, which has the form `/people/{username}/preferences`.